cdap-proto/pom.xml

```diff
@@ -51,6 +51,10 @@
       <artifactId>guava</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
```

Comment:
proto should be as light as possible, especially since it is sometimes used by other projects. Most of the time java's Collections.unmodifiableX() are good enough instead of guava's Immutable collections.

Reply:
done